medallion/exceptions.py

```diff
@@ -0,0 +1,46 @@
+# Medallion Custom Exceptions
+
+
+class MedallionError(Exception):
+    """base error class for Medallion
+
+    Args:
+        root_exception (Exception): Exception instance of root exception
+        desc (str): specific error description
+    """
+    def __init__(self, root_exception, desc=None):
```

Comment:
Many existing Exception classes expect a "message" as the first argument. I'm OK with doing it this way, since all of our exceptions are really wrappers around other exceptions that occur (`root_exception` is always required), but just wanted to point this out in case there's a better way to do it.

Reply:
I actually had it (desc, root_exception) first but felt desc was not a required parameter but root_exception was for the purposes of our custom exceptions, thus I flipped them. I can reverse it.